packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.component.js

```diff
@@ -0,0 +1,90 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import get from 'lodash/get';
+import classNames from 'classnames';
+import { CellMeasurer, CellMeasurerCache } from 'react-virtualized';
+
+import PureCollapsiblePanel from './PureCollapsiblePanel';
+import { getId, getRowData } from '../utils/gridrow';
+
+import withListGesture from '../../Gesture/withListGesture';
+import css from './RowCollapsiblePanel.scss';
+
+const cache = new CellMeasurerCache({ fixedWidth: true });
+const options = {
+	deferredMeasurementCache: cache,
+	rowHeight: cache.rowHeight,
+};
+
+/**
+ * Row renderer that displays a Collapsible Panel
+ */
+class RowCollapsiblePanel extends React.Component {
+	constructor(props) {
+		super(props);
+		this.onToggle = this.onToggle.bind(this);
+	}
+	onToggle(event, measure) {
+		const { parent, index } = this.props;
+		if (parent.props.onRowClick) {
+			parent.props.onRowClick({ event, rowData: { ...getRowData(parent, index), index } });
+			setTimeout(measure, 0);
+		}
+	}
+	render() {
+		const { className, index, onKeyDown, parent, style } = this.props;
+
+		const parentId = getId(parent);
+		const id = parentId && `${parentId}-${index}`;
+		const rowData = getRowData(parent, index);
+
+		return (
+			<CellMeasurer
+				cache={options.deferredMeasurementCache}
+				columnIndex={0}
+				key={this.props.index}
+				parent={this.props.parent}
+				rowIndex={index}
+			>
+				{({ measure }) => (
+					// eslint-disable-next-line jsx-a11y/no-static-element-interactions
+					<div
+						className={classNames(css['tc-collapsible-row'], rowData.className, className)}
```

Comment:
So I would add it here (the global class) for customization: `tc-collapsible-row`